src/Pages/TranslationManagerPage.php

```diff
@@ -89,6 +87,11 @@ public static function getNavigationLabel(): string
         return trans('filament-translation-manager::messages.title');
     }
 
+    public static function getNavigationIcon(): ?string
+    {
+        return config('filament-translation-manager.navigation_icon');
+    }
```

Comment:
Please add a default value here for the people that do not upgrade their config file.

Reply:
Sure. I'll update it

Reply:
Sure. I'll update it.